sachungo/movies
# Feature add movie cast

### Which issue does this PR relate to?

https://www.pivotaltracker.com/story/show/160052995

### What are the changes implemented in the PR?



- Added the cast information of a movie to movie detail's page
    - The cast information will be fetched together with the movie info when you  land on `/movies/:movie_id` page as the entry point to the app
    - The cast information will be fetched later when you click on a movie from the homepage to see it's detail

### How should the implementation be QA'd?



- Visit the homepage(by clicking `View deployment` button in the comment), click on a movie to be taken to `/movies/:movie_id` page
   - Confirm that you see the loading spinner when the cast is fetched
   - Verify that you see the cast information displayed as given in the screenshot once the fetch process is done
- Go back to the homepage, click a different movie and verify that you get a different cast from the one of the first movie to be clicked
- Copy the url of the movie info page you are at, open incognito browser, and load the url you had copied
   - Verify that the cast information is loaded together with the data

### Anything else to note


None

### Screenshots and/or gifs


![screen recording 2018-08-28 at 05 06 pm](https://user-images.githubusercontent.com/12892171/44728863-1cf7c500-aae6-11e8-8889-e29a2d8a6325.gif)

---

**Mobile**
![image 2018-08-28 at 5 06 31 pm](https://user-images.githubusercontent.com/12892171/44728898-3731a300-aae6-11e8-9dce-6406344ab9b1.png)

---

**Desktop**
![image 2018-08-28 at 5 06 03 pm](https://user-images.githubusercontent.com/12892171/44728919-444e9200-aae6-11e8-8446-65d87f10502b.png)
